patches/0076-Patch-computed-only-tags-for-iot-billing_group.patch

```diff
@@ -5,7 +5,7 @@ Subject: [PATCH] Patch computed only tags for iot/billing_group
 
 
 diff --git a/internal/service/iot/billing_group.go b/internal/service/iot/billing_group.go
-index 90d64baff5..dd84f3acb3 100644
+index 90d64baff5a..dd84f3acb33 100644
```

Comment:
Would be lovely if commits were not recorded and wouldn't churn, not sure if feasible though.